.npmignore

```diff
@@ -0,0 +1 @@
+package.json-e
```

Comment:
This file some why appears after run: sed -i -e 's/^  "version": "[^"]*"/  "version": "#{version}"/g' package.json with old content of file.
I added it to avoid this file from publication